visualization/src/pcl_visualizer.cpp

```diff
@@ -1503,6 +1503,39 @@ pcl::visualization::PCLVisualizer::getPointCloudRenderingProperties (int propert
   }
   return (true);
 }
+/////////////////////////////////////////////////////////////////////////////////////////////
+bool
+pcl::visualization::PCLVisualizer::getPointCloudRenderingProperties (RenderingProperties property, double &val1, double &val2, double &val3, const std::string &id)
```

Comment:
Split each argument to one line each. This line is excessively long.

Reply:
Done.